### PR TITLE
[DO] Adding entry for DO changelog retrospectively.

### PR DESCRIPTION
**Pull request by Oxyjun**
### Summary

Closes out https://github.com/cloudflare/cloudflare-docs/pull/16590

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by marciocloudflare — APPROVED** [review]

File: src/content/changelogs/durable-objects.yaml [hunk 1]
```diff
@@ -19,6 +19,13 @@ entries:
 
       During the initial beta, Storage API billing is not enabled for Durable Object classes using SQLite storage backend. SQLite-backed Durable Objects will incur [charges for requests and duration](/durable-objects/platform/pricing/#billing-metrics). We plan to enable Storage API billing for Durable Objects using SQLite storage backend in the first half of 2025 after advance notice with the following [pricing](/durable-objects/platform/pricing/#sqlite-storage-backend).
 
+  - publish_date: "2024-09-07"
+    title: New error message for overloaded Durable Objects
+    description: |-
+      Introduced a new overloaded error message for Durable Objects: "Durable Object is overloaded. Too many requests for the same object within a 10 second window."
```
Comment (irvinebroque) [inline src/content/changelogs/durable-objects.yaml:25]: Should this be added here? https://developers.cloudflare.com/durable-objects/observability/troubleshooting/#durable-object-is-overloaded

Comment (Oxyjun) [reply src/content/changelogs/durable-objects.yaml:25]: This is an update to the changelog - this PR was spun off an old PR, so I thought we were adding this in (retrospectively) to be comprehensive?

In the initial PR, the changelog entry was accidentally written for Workers - all I've done is to move it to the DO changelog.

Comment (vy-ton) [reply src/content/changelogs/durable-objects.yaml:25]: @Oxyjun we should add the DO changelog entry. Brendan is suggesting that we also document the new error in our overload troubleshooting guide, which I agree we should do

Comment (Oxyjun) [reply src/content/changelogs/durable-objects.yaml:25]: Ah got it. Documented the new error message along with the rest of the overloaded error messages.

**Review by vy-ton — APPROVED** [review]

**Pull request merged**